### ISSUE TITLE
Menu button registers every click

### ISSUE THREAD

**Issue by pablo3x6** [fluttercommunity/backdrop#45]
**Describe the bug**

**To Reproduce**
1. Upgrade to latest flutter
2. Click anywhere
3. Menu opens after every click anywhere
4. Be sad

**Expected behavior**
Whatever is in the container. 

**Screenshots**
[https://i.imgur.com/eGnDTjW.gif](url)
**Desktop (please complete the following information):**
 - OS: Windows
 - Browser Firefox
 - Version 10

**Smartphone (please complete the following information):**
 - Device: Anroid Pixel 3a
 - OS: Android 9
 - Browser default
 - Version 29

**Additional context**
Would appreciate if this could be fixed.

**Comment by WieFel** [comment]
Thanks for writing this issue @pablo3x6!
This bug is the same as in #39. It has already been fixed on the master branch of this repo.

However, the fix was not yet published on dart pub. You could meanwhile use the git repo as a dependency like so (in your `pubspec.yaml`):
```dart
dependencies:
  backdrop:
    git:
      url: git@github.com:fluttercommunity/backdrop.git
```

**Comment by daadu** [comment]
The fix for it has landed in `v0.4.2` on pub.dev
